randomizer/Lists/CBLocations/CreepyCastleCBLocations.py

```diff
@@ -2051,7 +2060,7 @@
         konglist=[Kongs.lanky],
         region=Regions.Mausoleum,
         vanilla=True,
-        logic=lambda l: l.sprint,
+        logic=lambda l: l.grape and (l.sprint or l.generalclips or l.phasewalk),
```

Comment:
Good find, but the gun requirement for the balloon is solved in addCollectible in Logic.py. safe to put (grape or sprint) or clips or phasewalk, or even remove it. your choice